### PR TITLE
Ideas for standard security disclaimer

### PR DESCRIPTION
**Pull request by ValuedMammal**
I imagine there's plenty of examples on the web for infosec disclaimers, but we could use the opportunity to invent our own - make it short and sweet, bitcoin focused, and characteristically BDK.

### CODE REVIEW

File: docs/cookbook/descriptors.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 BDK is a descriptor-first library. This page explores how to build them and how they interact with other standards like BIP-39 recovery phrases.
 
 !!!danger
-    TODO: Standard notice of danger of working with private keys, generating entropy, etc.
+    The keys and descriptors used in **The Book of BDK** are for illustration purposes only; **UNDER NO CIRCUMSTANCES** should any of the keys or descriptors containing private data be used for real money. Best practice for generating entropy is to do so in a controlled environment where information won't be exposed to untrusted networks.
```
Comment (storopoli) [inline docs/cookbook/descriptors.md:5]: Also mention that you should rely on a [cryptographically secure pseudorandom number generator (CSPRNG)](https://en.wikipedia.org/wiki/Cryptographically_secure_pseudorandom_number_generator)

Note that all the milksad.info exposed users where following the directives above, but Libbitcoin Explorer (`bx`) used a non-CS PRNG and with only 32 bits of entropy based on system time.

Comment (ValuedMammal) [reply docs/cookbook/descriptors.md:5]: That's a great point

**Review by storopoli — APPROVED** [review]
ACK 7ce8ae8253c70fd588bd5440678f24dcea2c6950

**Review by thunderbiscuit — APPROVED** [review]
ACK 7ce8ae8253c70fd588bd5440678f24dcea2c6950. Thanks!

**Pull request merged**